pyftdi/spi.py

```diff
@@ -583,6 +597,17 @@ def gpio_all_pins(self):
         with self._lock:
             return mask & ~self._spi_mask
 
+    @property
+    def _cs_bits(self):
```

Comment:
Should be public I believe

Reply:
This refers to an old version. I had this property at first but realized that I don't need it, so I removed it in a later commit (there are currently three commits). Do you prefer to have it as a property?

Reply:
I think for the port it would be nice to have a boolean that reflects its configuration.
For the controller I'm not sure it is worth it.